addons/sale/wizard/sale_order_discount.py

```diff
@@ -107,6 +107,7 @@ def _create_discount_lines(self):
                 discounted_price = line.price_unit * (1 - (line.discount or 0.0)/100)
                 total_price_per_tax_groups[line.tax_id] += (discounted_price * line.product_uom_qty)
 
+            discount_dp = self.env['decimal.precision'].precision_get('Discount') or 2
```

Comment:
```suggestion
            discount_dp = self.env['decimal.precision'].precision_get('Discount')
```
The `precision_get` method already returns `2` as default value, so you can leave out that part: https://github.com/odoo/odoo/blob/28f2b7f48f5279bfe7b49323ba8f7fc306f57b4c/odoo/addons/base/models/decimal_precision.py#L30-L34

Reply:
Indeed. As I encountered an issue in another ticket about unwanted tags removal this was a fallback value if the `Discount` precision can't be found.